src/mlpack/tests/CMakeLists.txt

```diff
@@ -78,6 +79,8 @@ add_executable(mlpack_test
   quic_svd_test.cpp
   radical_test.cpp
   random_forest_test.cpp
+  random_test.cpp
+  random_forest_test.cpp
```

Comment:
Looks like there is a merge artifact, if you remove the duplication travis should build the PR again.